src/compiler/Model.ts

```diff
@@ -232,17 +232,17 @@ export class Model {
   }
 
   /**
-   * @return Config value from the spec, or a default value if unspecified.
+   * Get the spec configuration.
    */
-  public config(name: string) {
-    return this._spec.config[name];
+  public config() {
+    return this._spec.config;
   }
 
   /**
-   * @return Cell config value from the spec, or a default value if unspecified.
+   * Get the spec configuration for cells.
    */
-  public cellConfig(name: string) {
-    return this._spec.config.cell[name];
+  public cellConfig() {
```

Comment:
Actually, should we just remove `scene|cellConfig()` and use `config().scene|cell` ?